tests/protocol/mockwiredata.py

```diff
@@ -286,15 +291,17 @@ def mock_http_post(self, url, *args, **kwargs):  # pylint: disable=unused-argume
         resp.read = Mock(return_value=content.encode("utf-8"))
         return resp
 
-    def mock_http_put(self, url, *args, **kwargs):  # pylint: disable=unused-argument
-        content = None
+    def mock_http_put(self, url, data, **kwargs):  # pylint: disable=unused-argument
```

Comment:
NIT: pylint error.
If the variable is not being used, please rename it to `**_` or something